test/acceptance/cli-test.js

```diff
@@ -1644,6 +1657,668 @@ describe('ember-template-lint executable', function () {
+        `);
+      });
+
+      it('should set todo to warn if warnDate has expired via option', async function () {
+        project.setConfig({
+          rules: {
+            'no-bare-strings': true,
+          },
+        });
+        project.write({
+          app: {
+            templates: {
+              'application.hbs': '<div>Bare strings are bad...</div>',
+            },
+          },
+        });
+
+        await run(['.', '--update-todo', '--todo-days-to-warn', '5'], {
+          env: {
+            TODO_CREATED_DATE: subDays(new Date(), 10).toJSON(),
+          },
+        });
+
+        const result = await run(['.']);
+
+        expect(result.exitCode).toEqual(0);
+        expect(result.stdout).toMatchInlineSnapshot(`
+          "app/templates/application.hbs
+            1:5  warning  Non-translated string used  no-bare-strings
+
+          ✖ 1 problems (0 errors, 1 warnings)"
+        `);
+      });
+
+      it('should set todo to warn if warnDate has expired but errorDate has not', async function () {
+        project.setConfig({
+          rules: {
+            'no-bare-strings': true,
+          },
+        });
+        project.write({
+          app: {
+            templates: {
+              'application.hbs': '<div>Bare strings are bad...</div>',
+            },
+          },
+        });
+
+        project.writeTodoConfig({
+          warn: 5,
+          error: 10,
+        });
+
+        await run(['.', '--update-todo'], {
+          env: {
+            TODO_CREATED_DATE: subDays(new Date(), 7).toJSON(),
+          },
+        });
+
+        const result = await run(['.']);
+
+        expect(result.exitCode).toEqual(0);
+        expect(result.stdout).toMatchInlineSnapshot(`
+          "app/templates/application.hbs
+            1:5  warning  Non-translated string used  no-bare-strings
+
+          ✖ 1 problems (0 errors, 1 warnings)"
+        `);
+      });
+
+      it('should set todo to error if errorDate has expired via config', async function () {
+        project.setConfig({
+          rules: {
+            'no-bare-strings': true,
+          },
+        });
+        project.write({
+          app: {
+            templates: {
+              'application.hbs': '<div>Bare strings are bad...</div>',
+            },
+          },
+        });
+
+        project.writeTodoConfig({
+          error: 5,
+        });
+
+        await run(['.', '--update-todo'], {
+          env: {
+            TODO_CREATED_DATE: subDays(new Date(), 10).toJSON(),
+          },
+        });
+
+        const result = await run(['.']);
+
+        expect(result.exitCode).toEqual(1);
+        expect(result.stdout).toMatchInlineSnapshot(`
+          "app/templates/application.hbs
+            1:5  error  Non-translated string used  no-bare-strings
+
+          ✖ 1 problems (1 errors, 0 warnings)"
+        `);
+      });
+
+      it('should set todo to error if errorDate has expired via env var', async function () {
```

Comment:
In this test you're testing it via CLI options, not env vars.

Reply:
Fixed.